superbit_lensing/medsmaker/superbit/astro_config/piff.config

```diff
@@ -107,10 +107,10 @@ select:
     min_snr: 20
 
     # Reserve 20% of the stars for diagnostics
-    reserve_frac: 0.1
+    reserve_frac: 0.15
 
     # If you want the reserve choice consistent across runs, set this to something.
-    seed: 57187153
+    seed: 
```

Comment:
Perhaps; it doesn't seem like very much work to add it to the list of required seeds, though it may be tedious to ensure that the seed is different for each exposure but repeatable between runs.

Reply:
Right, we'd have to add a whole extra config produced by `job-configs` with a separate seed sequence generation for each exposure. I think we should put this on the maybe-to-do-list for the optics PSF run, but isn't a huge priority for the gauss PSF validation run as I doubt we'll need to rerun anything again